### ISSUE TITLE
compile `make all` error at `FURY3`

### ISSUE THREAD

**Issue by meoso** [emuflight/EmuFlight-Butter-Varient#29]
compiling `make all` fails at the `FURY3`.
i don't need fury3, i just want a succesful `make all` to complete.

```shell
[...]
./src/main/fc/config.c: In function 'validateAndFixConfig':
./src/main/fc/config.c:305:53: error: 'FAILSAFE_PROCEDURE_GPS_RESCUE' undeclared (first use in this function); did you mean 'FAILSAFE_PROCEDURE_DROP_IT'?
         if (failsafeConfig()->failsafe_procedure == FAILSAFE_PROCEDURE_GPS_RESCUE) {
                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                     FAILSAFE_PROCEDURE_DROP_IT
./src/main/fc/config.c:305:53: note: each undeclared identifier is reported only once for each function it appears in
%% piniobox.c 
make[2]: *** [Makefile:308: obj/main/FURYF3/fc/config.o] Error 1
make[2]: *** Waiting for unfinished jobs....
%% (size optimised) serial.c 
%% statusindicator.c 
make[2]: Leaving directory '/home/USERNAME/Downloads/EmuFlight-Butter-Varient'
make[1]: *** [Makefile:415: binary] Error 2
make[1]: Leaving directory '/home/USERNAME/Downloads/EmuFlight-Butter-Varient'
make: *** [Makefile:359: FURYF3] Error 2
```

& new issue options say betaflight
& this was the "closest" category to open the issue

**Comment by Quick-Flash** [comment]
If you grab the latest code this should no longer be an issue. We are no longer supporting f3 targets and as such we have removed all the f3 targets from our code. You should be able to do a successful make all now!
